database/N/Nicole/Flieg nicht so hoch, mein kleiner Freund/Flieg nicht so hoch, mein kleiner Freund

```diff
@@ -0,0 +1,49 @@
+An dem großen gelben Fluss da saß ein Mann
+Dass er traurig war, das sah man ihm gleich an
+Auf dem Baume neben ihm, saß ein Vogel und es schien
+Dieser Mann singt sein Lied nur für ihn
+
+Flieg nicht so hoch mein kleiner Freund
+Die Sonne brennt dort oben heiß
+Wer zu hoch hinaus will der ist in Gefahr
+Flieg nicht so hoch mein kleiner Freund
+Glaub' mir ich mein es gut mit dir
+Keiner hilft dir dann, ich weiß es ja
+Wie's damals bei mir war
+
+Über'm Fluss kam die Nacht schon angekrochen
+Und die beiden saßen noch am gleichen Platz
+Und er sang vom roten Sand und dem großen fernen Land
+Und vom Glück das er leider niemals fand
+
+Flieg nicht so hoch mein kleiner Freund
+Die Sonne brennt dort oben heiß
+Wer zu hoch hinaus will der ist in Gefahr
+Flieg nicht so hoch mein kleiner Freund
+Glaub' mir ich mein es gut mit dir
+Keiner hilft dir dann, ich weiß es ja
+Wie's damals bei mir war
+
+Und am Morgen stand der Alte an der Straße
+Und er winkte doch die Autos fuhr vorbei
+Als er dann den Tränen nah, einen toten Vogel sah
+Glaubte er, dass dies sein Freund von gestern war
+
+Flieg nicht so hoch mein kleiner Freund
+Die Sonne brennt dort oben heiß
+Wer zu hoch hinaus will der ist in Gefahr
+Flieg nicht so hoch mein kleiner Freund
+Glaub' mir ich mein es gut mit dir
+Keiner hilft dir dann, ich weiß es ja
+Wie's damals bei mir war
+
+
+______________________________________________________________________________
```

Comment:
```suggestion
_______________________________
```

Reply:
What's wrong here @snshn ? The line is too long?